packages/twenty-front/src/pages/settings/serverless-functions/SettingsServerlessFunctionDetail.tsx

```diff
@@ -169,11 +178,19 @@ export const SettingsServerlessFunctionDetail = () => {
     }
     setActiveTabId('test');
   };
+  const isAnalyticsEnabled = useRecoilValue(isAnalyticsEnabledState);
+
+  const isAnalyticsV2Enabled = useIsFeatureEnabled('IS_ANALYTICS_V2_ENABLED');
 
   const tabs = [
     { id: 'editor', title: 'Editor', Icon: IconCode },
     { id: 'test', title: 'Test', Icon: IconTestPipe },
     { id: 'settings', title: 'Settings', Icon: IconSettings },
+    ...(isAnalyticsEnabled && isAnalyticsV2Enabled
+      ? [
+          { id: 'monitoring', title: 'Monitoring', Icon: IconBox }, //IconGauge from figma is not in twenty ui:(
```

Comment:
You can add any tabler icon to twenty-ui :). I think you just need to add it to `TablerIcons.ts` in twenty-ui